### PR TITLE
Updated push and pull task release notes 01/11

### PR DESCRIPTION
**Pull request by MaryanneNjeri**

### CODE REVIEW

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v5.1.0 - January, 07 2022
+* Supporting task in other sovereign clouds.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md:8]: Added support for sovereign clouds such as Azure Government and Azure China.

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v3.1.0 - January, 07 2022
+* Supporting task in other sovereign clouds.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:6]: Ditto

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v5.1.0 - January, 07 2022
+* Added support for sovereign clouds such as Azure Government and Azure China.
+* Updated task input from **App Configuration Name** to **App Configuration Endpoint**.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md:9]: This is a breaking change, right? We should mark it as breaking change as we did before. Any instructions we could provide for users to upgrade from v4.* to v5.*?

Comment (MaryanneNjeri) [reply releaseNotes/AzureDevOpsPipelineExtension.md:9]: Yes, we should mark it as a breaking change, let me update that. For the instructions, how about?
**Before**: Task input was **App Configuration Name**
**After**: Task input updated to App Configuration endpoint, the  **App Configuration Endpoint** can be gotten from the App configuration store overview page.

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 4]
```diff
@@ -5,8 +5,13 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
 ### v5.1.0 - January, 07 2022
-* Added support for sovereign clouds such as Azure Government and Azure China.
-* Updated task input from **App Configuration Name** to **App Configuration Endpoint**.
+* **Breaking change**: 
+With the added support of sovereign clouds such as Azure Government and Azure China, the task inputs were updated.
+  **Before**:
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md:10]: An empty line is needed before this or it will be merged in the same line with the bullet point.

**Review by zhenlan — APPROVED** [review]

**Pull request merged**